tests/rpc/test_rpc_apiserver.py

```diff
@@ -974,7 +984,8 @@ def test_api_whitelist(botclient):
     }
 
 
-def test_api_forcebuy(botclient, mocker, fee):
+@pytest.mark.parametrize('is_short', [True, False])
+def test_api_forcebuy(botclient, mocker, fee, is_short):
```

Comment:
This test (for short) is testing the wrong thing.

We currently don't have a way to send a "/forceshort" to the bot - so this should not be possible.

I know the test works, and that's only because you cheat the `is_short` value into the mock response - which in reality is a method call, which will never receive the "should be short" notice.
So the test is making it seem like you can force-enter shorts - which you really can't at this moment.

I'd suggest to remove changes to this test again - as we'll need to come up with a proper strategy on how to handle "forced short entries" - which i think we should handle in a separate PR, as it'll not be a 2liner (probably, anyway).